tests/backend/api/test_health_config.py

```
@@ -0,0 +1,302 @@
+
+def test_detailed_config_health_endpoint(client, mock_validation_success):
+    """Test detailed config health endpoint."""
+    with patch("apps.backend.api.health.settings") as mock_settings:
+        mock_settings.APP_VERSION = "1.0.0"
+        mock_settings.ENVIRONMENT = "development"
+
+        # Access app state directly
+        from apps.backend.main import app
+
+        app.state.validation_report = mock_validation_success
+        app.state.start_time = datetime.now().timestamp()
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)